Fix logging in the api server, add relayer documentation. (#87)
Concordium/eth-ccd-bridge: relayer/CHANGELOG.md

## Unreleased changes

## 1.0.2

> - Fix logging in the api server. The logging should have correct contexts for
>   requests now.

Author: ```suggestion
- Fix logging in the api server. The logging has correct contexts for requests now.
```
